snippets/chainAsync.md

```diff
@@ -4,10 +4,12 @@ Chains asynchronous functions.
 
 Loop through an array of functions containing asynchronous events, calling `next` when each asynchronous event has completed.
 
+The tenerary function checks the next function exists before calling it, otherwise it will exit.
+
 ```js
 const chainAsync = fns => {
   let curr = 0;
-  const next = () => fns[curr++](next);
+  const next = () => (fns[++curr] ? fns[curr](next) : false);
```

Comment:
Hey @tcmal 👋 😄 

Thanks for submitting a PR

However I believe your code doesn't work as expected and should leave as it. As stated by @Chalarangelo 

```js
const chainAsync = fns => {
  let curr = 0;
  const next = () => (fns[++curr] ? fns[curr](next) : false);
  next();
};
```

The following doesn't run as expected
```js
chainAsync([
  next => {
    console.log('0 seconds');
    setTimeout(next, 1000);
  },
  next => {
    console.log('1 second');
  }
]);
```

`1 second` is printed to the console automatically instead of working as expected

Reply:
Ah ok, think I misunderstood original review. Have commited change that appears to fix this on my end.